src/pages/Integrations/List/Page.tsx

```diff
@@ -35,8 +36,12 @@ export const IntegrationsListPage: React.FunctionComponent = () => {
     const integrationRows = useIntegrationRows(integrations);
     const history = useHistory();
 
+    const [activateModal,updateModal] = useState(false);
+    const [model,updateModel] = useState([integrationRows.rows]);
+
     const onAddIntegration = React.useCallback(() => {
-        history.push(linkTo.addIntegration());
+        // history.push(linkTo.addIntegration());
+        updateModal(true);
```

Comment:
I think that you are confusing how the `useState` hook works. What exactly are you trying to achieve here?

Reply:
Nevermind, I was actually confused by the name. (updateModal / updateModel)

Reply:
To activate the Modal within the click action handler...